src/common-components/tests/__snapshots__/ThirdPartyAuthAlert.test.jsx.snap

```diff
@@ -21,26 +21,33 @@ exports[`ThirdPartyAuthAlert should match login page third party auth alert mess
 `;
 
 exports[`ThirdPartyAuthAlert should match register page third party auth alert message snapshot 1`] = `
-<div
-  className="fade alert-content alert-success mt-n2 alert show"
-  id="tpa-alert"
-  role="alert"
->
+Array [
```

Comment:
Remind me: Had we not removed snapshot tests from authn?

Reply:
We removed it from main workflows but these smaller components still have few snapshot tests.